tests/unit-tests/bacs-accounts.php

```diff
@@ -148,4 +148,192 @@ private function get_settings( $gateway_class ) {
 		}
 		return $settings;
 	}
+
+	/**
+	 * Test updating bacs gateway with no account data supplied.
+	 */
+	public function test_update_bacs_payment_gateway_with_no_account_data() {
+		wp_set_current_user( $this->user );
+
+		$request = new WP_REST_Request( 'POST', '/wc/v3/payment_gateways/bacs' );
+		$request->set_body_params( array(
+			'settings' => array(
+				'title' => 'Pay with bank transfer!',
+			),
+		) );
+		$response = $this->server->dispatch( $request );
+		$bacs = $response->get_data();
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertEquals( 'Pay with bank transfer!', $bacs['settings']['title']['value'] );
+		$this->assertEquals( array(), $bacs['settings']['accounts']['value'] );
+	}
+
+	/**
+	 * Test updating bacs gateway with account data supplied.
+	 */
+	public function test_update_bacs_payment_gateway_with_account_data() {
+		wp_set_current_user( $this->user );
+
+		$new_accounts = array(
+			array(
+				'account_name'   => 'scrooge mcduck',
+				'account_number' => 'bling',
+				'bank_name'      => 'disney bank',
+				'sort_code'      => '111',
+				'iban'           => '222',
+				'bic'            => 'wat',
+			),
+			array(
+				'sort_code'      => '111',
+				'iban'           => '222',
+				'bic'            => 'wat',
+				'account_name'   => 'something else',
+				'account_number' => '123134844584548',
+				'bank_name'      => 'blaaaaahhhh',
+			)
+		);
+
+		$accounts_request = new WP_REST_Request( 'POST', '/wc/v3/payment_gateways/bacs' );
+		$accounts_request->set_body_params( array(
+			'settings' => array(
+				'accounts' => $new_accounts,
+			),
+		) );
+		$accounts_response = $this->server->dispatch( $accounts_request );
+		$bacs_accounts = $accounts_response->get_data();
+		$this->assertEquals( 200, $accounts_response->get_status() );
+		$this->assertEquals( $new_accounts, $bacs_accounts['settings']['accounts']['value'] );
+	}
+
+	/**
+	 * Test updating bacs gateway with account data supplied.
```

Comment:
Minor: Update description to include that the title / non account data is also being supplied.